dev_guide/builds/triggering_builds.adoc

```diff
@@ -405,3 +412,26 @@ Configuration change triggers currently only work when creating a new
 `BuildConfig`. In a future release, configuration change triggers will also be
 able to launch a build whenever a `BuildConfig` is updated.
 ====
+
+[[setting-triggers]]
+=== Setting Webhook Triggers Manually
+
+Triggers on configurations that are not created automatically can be generated
+with `oc set triggers`. For example, to set a GitHub webhook trigger on a Build
+configuration, use:
```

Comment:
"Triggers can be added to and removed from build configurations with `oc set triggers`.  For example....."